ci-plan.md

```diff
@@ -0,0 +1,44 @@
+# Plan for adding embedded CI tests to `rust-lang/rust`
+
+* Goal: we will add target `thumbv7m-none-eabi` in stable Rust 2018 Edition
+  * [TODO] Is the target name right?
+
+## Tests not to add
+
+* `dist` target for `thumbv7m-none-eabi`
+  * `./x.py dist` for `thumbv7m-none-eabi` is already included in `dist-various-1` docker image; this is why we can do `rustup target add` for the target now.
+
+## Tests to add
+
+> TODO: Add more detail
+
+* Make sure known-good embedded projects still compile and link
+  * [???] We could use `cargotest` infra or alike.
+  * Crate `cortex-m` is a good start because it already compiles on beta now.
+  * `stm32f103xx-hal` includes `cortex-m` in dependency, so we'll use it later.
```

Comment:
:+1:, I agree `cortex-m` is a good place to start for now as it is supported by beta

For more candidates, I would suggest the `examples` folder: https://github.com/japaric/stm32f103xx-hal/tree/master/examples